assignments/objects.js

```diff
@@ -18,39 +18,89 @@ const example = {
 }
 
 // Write your intern objects here:
+const intern1 = {
+  "id": "1",
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+
+const intern2 = {
+  "id": "2",
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M",
+  speak: function () {
```

Comment:
if you do this 
```
"speak":  function() {
    return ("Hello, my name is Kennan!");
  }
```

then you can 

```
console.log(intern2.speak())
```

Try something similar to fix your multiply nums and console log that